sdks/python/apache_beam/examples/ml-orchestration/kfp/components/train/component.yaml

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: model training
+description: Train a pytorch model
+inputs:
+  - name: base_artifact_path
+    description: base path to store data
+    type: String
+  - name: preprocessed_dataset_path
+    description: path to the preprocessed dataset
+    type: String
+outputs:
+  - name: trained_model_path
+    description: trained model file
+    type: String
+implementation:
+  container:
+    image: <your-docker-registry/train-image-name:latest>
```

Comment:
It would be nice having this as a command line argument or environment variable.